sycl/test/basic_tests/buffer/buffer_reinterpret.cpp

```diff
@@ -0,0 +1,70 @@
+// RUN: %clangxx -fsycl %s -o %t.out
```

Comment:
```suggestion
// RUN: %clangxx -fsycl -fsyntax-only %s
```
We need only to check that compiler front-end do not report errors, right?